### PR TITLE
Parallel integration tests to speed up from ~2min to 10s

### PR DESCRIPTION
**Pull request by shifucun**
- Add t.Parallel() to all the integration tests.
- The TestChartConfigRankings ranking test have two for loops over place type (4) and chart config (>100) which takes about 60 seconds to run. Use goroutine to parallel at config level so the O(100) request can emit concurrently. Hence needs to sort the final result by chart title and updating the golden response.

---
This change is [Reviewable](https://reviewable.io/reviews/datacommonsorg/mixer/421)

### CODE REVIEW

File: test/integration/place_statvar_ranking_test.go [hunk 1]
```diff
@@ -114,59 +117,72 @@ func TestChartConfigRankings(t *testing.T) {
...
+					// Test main chart rankings
+					req := &pb.GetLocationsRankingsRequest{
+						PlaceType:    c.placeType,
+						WithinPlace:  c.parentPlace,
+						StatVarDcids: chart.StatsVars,
+						IsPerCapita:  len(chart.Denominator) > 0,
+					}
+					missingStatVars, err := getMissingStatVarRanking(client, req)
+					if err != nil {
+						t.Errorf("Error fetching rankings for chart %s: %s", chart.Title, c.placeType)
```
Comment (pradh) [inline test/integration/place_statvar_ranking_test.go:141]: nit: is the line too long?

**Review by pradh — APPROVED** [review]
Nice!

File: test/integration/place_statvar_ranking_test.go (hunk 1 above)
Comment (shifucun) [reply test/integration/place_statvar_ranking_test.go]: Move long lines to multi-lines

**Comment by shifucun** [comment]
thanks for quick review!

**Pull request merged**